app/src/main/java/org/kiwix/kiwixmobile/zim_manager/ZimManageActivity.kt

```diff
@@ -68,7 +65,7 @@ class ZimManageActivity : BaseActivity() {
     setUpToolbar()
     manageViewPager.run {
       adapter = mSectionsPagerAdapter
-      offscreenPageLimit = 2
+      offscreenPageLimit = mSectionsPagerAdapter.count - 1
```

Comment:
`mSectionsPagerAdapter` should be rename to `sectionsPagerAdapter`.

Reply:
done